Formula/lmod.rb

```diff
@@ -0,0 +1,64 @@
+class Lmod < Formula
+  desc "Lua-based environment modules system to modify PATH variable"
+  homepage "https://www.tacc.utexas.edu/research-development/tacc-projects/lmod"
+  url "https://github.com/TACC/Lmod/archive/7.7.16.tar.gz"
```

Comment:
7.7.15 is still marked as "latest release"

Reply:
I've asked a question on LMod, will see what they say or roll it back to 7.7.15 if there's no answer soon.

Reply:
7.7.18 is now officially released, using that.